website_blog_excerpt_img/README.rst

```diff
@@ -22,47 +52,59 @@ Usage
 
 To use this module, you need to:
 
-* Go to your blog.
+#. Go to your blog.
+#. Disable the cover image from *Customize > Blog Posts > Cover image*
+   to avoid having too many images, and possibly repeated ones.
+#. Enable the template from *Customize > Blog Posts >
+   Floating social share image* if it is not enabled by default.
+#. Your posts will now display a thumbnail of the chosen social share image.
 
-.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
-   :alt: Try me on Runbot
-   :target: https://runbot.odoo-community.org/runbot/186/12.0
+This module also will autoselect the 1st image in the blog content as
+the social share image, if there's no post cover and the publisher didn't
+choose a different one manually, from the *Promote > Optimize SEO* wizard.
 
-Known issues / Roadmap
-======================
+See it in action:
 
-* Right now, it will display the cover image if available, and an image
-  extracted from the post body otherwise. It would be nice to let the web admin
-  choose that, or even better to display a carousel with all images from the
-  post.
+.. figure:: https://user-images.githubusercontent.com/973709/56575735-72756a80-65be-11e9-9ecd-5a4b18fe5be1.gif
+   :alt: Instructions and explanation on what this module does
 
 Bug Tracker
 ===========
 
-Bugs are tracked on `GitHub Issues
-<https://github.com/OCA/website/issues>`_. In case of trouble, please
-check there if your issue has already been reported. If you spotted it first,
-help us smashing it by providing a detailed and welcomed feedback.
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/website/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/website/issues/new?body=module:%20website_blog_excerpt_img%0Aversion:%2012.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
 
 Credits
 =======
 
+Authors
+~~~~~~~
+
+* Grupo ESOC Ingeniería de Servicios
```

Comment:
Should we still keep this author? I don't think so.

Reply:
I think it's the most elegant. They still exist and they actually authored the module, so I see no reason for removing them. This is not "maintainers", after all.

Reply:
I don't agree. Here elegance has nothing to do. You have even rewritten almost everything and you are Tecnativa's employee, not Grupo ESOC's one. They already have the credit for previous versions.